tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -63,20 +64,88 @@ Object.Dai {
 }
 
 Object.Pipeline {
-	passthrough-capture.100 {
+	gain-capture.3 {
```

Comment:
I prefer to keep the index 100 that will align with the index below

Reply:
sure, thanks